calnex/api/api_test.go

```diff
@@ -760,3 +760,15 @@ func TestPushLicenseError(t *testing.T) {
 
 	require.Equal(t, originalLicense, uploadedLicense)
 }
+func TestReboot1(t *testing.T) {
```

Comment:
Why Reboot**1**?

Reply:
I am so sorry. I was just testing a couple of things together hence renamed functions but forgot to restore the name.Will do that